docroot/themes/humsci/humsci_basic/src/scss/components/_block-layout.scss

```diff
@@ -4,7 +4,11 @@
 
 .block-hs-blocks {
   padding: hb-calculate-rems(20px) hb-calculate-rems(18px);
-  border: $hb-menu-light-border;
+
+  @include hb-colorful {
```

Comment:
doesnt need to be inside the hb-colorful mixin as that happens in the hb-global-color mixin

Reply:
Maybe we should talk about whether the adjustment we want to make is deleting this @include, or moving it out of the mixin?  🤔